pkg/crc/machine/machine.go

```diff
@@ -224,6 +225,14 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		logging.InfoF("Login to the console with user: kubeadmin, password: %s", result.ClusterConfig.KubeAdminPass)
 	}
 
+	// Approve the node certificate.
+	crcOc := oc.GetOC(machineConfig.Name)
```

Comment:
just reading this fragment of code makes me wonder what it does:

`get OC for an named instance of CRC` ?... why is `oc` associated with an instance?